src/aks-preview/HISTORY.rst

```diff
@@ -27,6 +26,7 @@ Pending
 1.0.0b3
 +++++++
 * Change the format for az aks machine commands to separate the ipv4, ipv6 columns
+* Deprecate the alias "-r" of parameter --source-resource-id in `az aks trustedaccess rolebinding create`
```

Comment:
The change is actually released in 1.0.0b3
![image](https://github.com/Azure/azure-cli-extensions/assets/81607949/e30456b9-53a9-4b1d-8144-6bbe8292e50a)